docs/pages/database-access/guides/mongodb-atlas.mdx

```diff
@@ -225,6 +225,58 @@ certificate with `CN=alice` subject.
   Case matters so make sure to specify Common Name in the username with capital
   letters `CN=`.
 </Admonition>
+</TabItem>
+
+<TabItem label="AWS IAM authentication">
+Teleport will authenticate with MongoDB Atlas using [AWS IAM roles](https://www.mongodb.com/docs/atlas/security/passwordless-authentication/).
+
+### Configure Teleport AWS credentials
+
+You must provide the Teleport Database Service access to AWS credentials.
+
+(!docs/pages/includes/aws-credentials.mdx service="the Database Service"!)
+
+### Create MongoDB IAM role
+
+Create an AWS IAM role that will be used as a MongoDB user. In this guide
+example, we will use the name `teleport-access`.
+
+Under the Trusted entity type, choose "Custom trust policy". Edit the trust
```

Comment:
What does "Under the Trusted entity type" refer to? If we're referring to the AWS console, I think we should link the reader to the relevant page within the console. We should also bold the appropriate form field name.

Reply:
I've changed this section to include a how-to to create the IAM role.